mem0/memory/setup.py

```diff
@@ -3,8 +3,8 @@
 import uuid
 
 # Set up the directory path
-home_dir = os.path.expanduser("~")
-mem0_dir = os.path.join(home_dir, ".mem0")
+home_dir = os.environ.get("HOME_DIR") or os.path.expanduser("~")
```

Comment:
```suggestion
home_dir = os.path.expanduser("~")
```

Reply:
You can keep the `home_dir` as is actually. You can directly set the `MEM0_DIR` and it will work just fine.